secret_data_test.py

```diff
@@ -0,0 +1,25 @@
+import os
+import json
+import glob
+import subprocess
+
+"""
+This is a test that uses old/unpublished data files that we don't want to make
```

Comment:
I think it could be quite a bit of work, as these are deliberately large files. I'd actually suggest going the opposite direction, and trying to cover more data, taking advantage of the fact we can just include existing files. @kindly may have some thoughts on this.